### PR TITLE
Initial commit for JS DOM

### PR DESCRIPTION
**Pull request by Jigsaff**
### Document Object Model

[Demo](https://jigsaff.github.io/dom-api-task/)
[Code base](https://github.com/Jigsaff/dom-api-task)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@Jigsaff thank you for your submission.

This is how your PR title looks among other PRs' titles.
image

PR message template also says
image

**Comment by OleksiyRudenko** [comment]
Please also familiarize yourself with update instructions on this repo.

image

Please fix this.

**Comment by Jigsaff** [comment]
Fixed, please review.

File: submissions/Jigsaff/dom-api-task/app/app.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    if (activeElement) {
+        activeElement.classList.remove("header__link--decorated");
+    }
+    target.classList.add("header__link--decorated");
+};
+
+const handleHeaderClick = ({target}) => {
+    if (target.classList.contains("header__link")) {
+        const sectionName = target.textContent;
```
Comment (MaxKalashnyk) [inline submissions/Jigsaff/dom-api-task/app/app.js:116]: It's not a blocker for this task but you can toggle sections by using data attributes with some ids. And make checking by id, not by the textContent.

File: submissions/Jigsaff/dom-api-task/app/app.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        const section = backpacks.find(
+            (item) => item.name.toLowerCase() === sectionName.toLowerCase()
```
Comment (MaxKalashnyk) [inline submissions/Jigsaff/dom-api-task/app/app.js:118]: You can destructure your `item` object here. Also, I'd recommend using not so generic names for variables. Using the `item` variable name is not very informative

**Review by MaxKalashnyk — CHANGES_REQUESTED** [review]
Good job! Appreciate you clean code.
Check my comments, please.
Also, pay attention to the mobile styles of your app for future. Now I see that your app doesn't look so nice on moblie devices.

File: submissions/Jigsaff/dom-api-task/app/app.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        activeElement.classList.remove("header__link--decorated");
+    }
+    target.classList.add("header__link--decorated");
+};
+
+const handleHeaderClick = ({target}) => {
+    if (target.classList.contains("header__link")) {
+        const sectionName = target.textContent;
+        const section = backpacks.find(
+            ({name: backpackName}) => backpackName.toLowerCase() === sectionName.toLowerCase()
```
Comment (MaxKalashnyk) [inline submissions/Jigsaff/dom-api-task/app/app.js:118]: 👍🏻

**Review by MaxKalashnyk — APPROVED** [review]
LGTM!

**Pull request merged**